build/configs/artik053/tc/defconfig

```diff
@@ -1312,6 +1312,24 @@ CONFIG_SYSIO_ITC_GPIO=y
 CONFIG_EXAMPLES_TESTCASE_TASK_MANAGER_UTC=y
 # CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_UTC is not set
 # CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_ITC is not set
+# CONFIG_EXAMPLES_TESTCASE_ST_THINGS_STC is not set
+#CONFIG_ST_THINGS is not set
```

Comment:
I think, out of all the new configuration parameters, commenting CONFIG_EXAMPLES_TESTCASE_ST_THINGS_STC would be sufficient. By running "make menuconfig",  we should be able to select this configuration.

Reply:
ok

Reply:
1. Except CONFIG_EXAMPLES_TESTCASE_ST_THINGS_STC, all other newly added configuration flags can be uncommented. 

2. README file (apps/examples/testcase/ta_tc/st_things/README.txt) can be updated with the same information.

3. There should be a space after '#'.

Reply:
Please check the details below about the comments.

1. Other config flags will lead to compilation of IoTivity stack and dependent libraries. Also ST things frame work unnecessarily. This will not only increase the binary size also may lead to build errors because of dependents.
We think enabling other flags lead to binary size increase or build errors.
3. Yes fixed and added space.